docs/sql-ref-syntax-ddl-create-database.md

```diff
@@ -19,4 +19,39 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+Creates a database with the given name. If database with the same name exists an exception will be thrown.
+
+### Syntax
+{% highlight sql %}
+CREATE (DATABASE|SCHEMA) [IF NOT EXISTS] database_name
+  [COMMENT database_comment]
+  [LOCATION database_directory]
+  [WITH DBPROPERTIES (property_name=property_value, ...)]
```

Comment:
@dilipbiswal @gatorsmile 
I am wondering if we should define a standard for the syntax format. Should we have a white space between the braces, =, |, ... and the text? I currently put a white space because I saw a couple of sql references have these standard. I am OK with either having white space or no white space, but I am thinking we may need to define a standard and write out the standard explicitly in the umbrella jira so everybody can follow.  Also, shall we have 4 white space indentation for the 2nd line? I am ok with either 2 or 4, but prefer everybody to follow the same way.

Reply:
@huaxingao About while spacing , i am +1 on this. About the indentation on the 2nd line, i don't have a strong preference. @gatorsmile do you have a preference here ?

Reply:
Updated white spaces wherever required. Thanks